plugins/check_dbi.c

```diff
@@ -426,6 +426,7 @@ process_arguments (int argc, char **argv)
 			else
 				timeout_interval = atoi (optarg);
 
+			break;
```

Comment:
This looks really like a break to me, as 't' (timeout) and 'H' (host) probably have nothing in common to be parsed. ok.
Actually, good catch..